mlserver/parallel/utils.py

```diff
@@ -15,7 +15,8 @@ def configure_inference_pool(settings: Settings):
 
     # Set MP method
     try:
-        multiprocessing.set_start_method("spawn")
+        # force set to True to override the "fork" setting used in pytest
+        multiprocessing.set_start_method("spawn", force=True)
```

Comment:
Nice one! That must have been a hard one to debug. 

I've been trying to find out more info about the `force=` kwarg, but it doesn't seem to be listed in the Python docs. Do you know whether there's any caveat that we should be aware of?

Reply:
> Do you know whether there's any caveat that we should be aware of?

Thanks a lot for the review. Yes, this one was really tough to figure out :) Regarding caveats of `force=True`. It's undocumented. The main caveat is in possible IPC object leaks in case that for example `fork` was set first and multiprocessing actually created new processes and after that you change it to `spawn`. Then cleaning-up those previously (fork-ed) created processes can be impacted. But if you haven't used multiprocessing and it's only resetting the value for the first time, it should be fine.

Probably the cleanest way would be propagating the multiprocessing context for all multiprocessing interactions, i.e.

```
ctx = mp.get_context('spawn')
q = ctx.Queue()
p = ctx.Process(target=foo, args=(q,))
```

then you can have several contexts `ctx` with different start methods in your python program

Reply:
Got it. Thanks for that extra context @lada-kunc!

Based on that, since MLServer is ran as a standalone process, it should be alright to keep the `force=` kwarg as-is. I agree that using `ctx` would probably be cleaner, but that's out of the scope of this PR. That's a good one to keep in mind though!